### DISCUSSION TITLE
Inquiry on Modifying FDTD Update Equations and Dielectric Constants in openEMS

### DISCUSSION THREAD

**Discussion by Derek-art** [thliebig/openEMS-Project#215] [Q&A Usage]
I have a question regarding the modification of the update equations in the FDTD solver in openEMS. Specifically, I need to modify the dielectric constant in these equations. Does openEMS provide an interface for modifying the FDTD update equations directly, or do I need to modify the .dll files to achieve this? If modification of the .dll files is required, could you please provide guidance on how to proceed?

**Comment by gadiLhv** [comment]
Hey @Derek-art 

Your question is unclear. What do you mean by "modify the dielectric constant"?

The dielectric constant (and the conductivity) changes along the predefined geometry. What did you mean?

**Reply by Derek-art** [reply]
I mean 
The permittivity is a function of time and space

**Reply by gadiLhv** [reply]
You mean:

A. That you want to change the permittivity along time?
B. You want a frequency dependant permittivity property?
